alpha/quarkus-chart/README.md

```diff
@@ -15,25 +15,15 @@ build:
 and apply by passing `--values $VALUES_FILE`.
 
 ## Values
-This section describes the Values used to configure this chart.
-
-### Required Values 
-Below shows the required values to install this chart.
-
-| Value | Description | Default | Additional Information |
-| ----- | ----------- | ------- | ---------------------- |
-| `build.uri` | Git URI that references your git repo | - | **REQUIRED** |
-
-### All Values
 Below is a table of each value used to configure this chart.
 
 | Value | Description | Default | Additional Information |
 | ----- | ----------- | ------- | ---------------------- |
 | `image.name` | Name of the image you want to build/deploy | Defaults to the Helm release name. | The chart will create/reference an [ImageStream](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html) based on this value. |
 | `image.tag` | Tag that you want to build/deploy | `latest` | The chart will create/reference an [ImageStreamTag](https://docs.openshift.com/container-platform/4.6/openshift_images/image-streams-manage.html#images-using-imagestream-tags_image-streams-managing) based on the name provided |
 | `build.enabled` | Determines if build-related resources should be created. | `true` | Set this to `false` if you want to deploy a previously built image. Leave this set to `true` if you want to build and deploy a new image. |
-| `build.uri` | Git URI that references your git repo | - | **REQUIRED** |
-| `build.ref` | Git ref containing the application you want to build | master | - |
+| `build.uri` | Git URI that references your git repo | https://github.com/deweya/quarkus-getting-started | This value defaults to a sample application. Be sure to override this if you want to build and deploy your own application. |
```

Comment:
How about we create a new org called something like `helm-quarkus` and move it there?

Reply:
We need something more official definitely

Reply:
Updated to use https://github.com/redhat-developer-helm-quickstarts/quarkus-getting-started